.audit-harness/scripts/escape-scan.sh

```
@@ -0,0 +1,171 @@
+#   coverage.line: 80
+#   coverage.branch: 70
+#   mutation.kill_rate: 70
+COVERAGE_LINE_FLOOR=80
+COVERAGE_BRANCH_FLOOR=70
+MUTATION_FLOOR=70
+TESTING_MD="$ROOT/tests/TESTING.md"
+if [[ -f "$TESTING_MD" ]]; then
+  v=$(grep -Ei '^\s*coverage\.line\s*:' "$TESTING_MD" | head -1 | sed -E 's/.*:\s*([0-9]+).*/\1/')
+  [[ -n "$v" ]] && COVERAGE_LINE_FLOOR="$v"
+  v=$(grep -Ei '^\s*coverage\.branch\s*:' "$TESTING_MD" | head -1 | sed -E 's/.*:\s*([0-9]+).*/\1/')
+  [[ -n "$v" ]] && COVERAGE_BRANCH_FLOOR="$v"
```

Shellcheck (0.11.0)

[warning] COVERAGE_BRANCH_FLOOR appears unused. Verify use (or export if used externally).

(SC2034)